support-frontend/assets/components/subscriptionCheckouts/threeTierTerms.tsx

```diff
@@ -27,57 +28,38 @@ const termsLink = (linkText: string, url: string) => (
 
 export default function ThreeTierTerms(props: {
 	paymentFrequency: 'month' | 'year';
+	manageLink: EmotionJSX.Element;
```

Comment:
does this need to be a prop? It seems to be unchanging in both places its passed

Reply:
Ah fair enough, separate file would be the easiest way but doesn't seem worthwhile

Reply:
I've extracted it to a separate file, I think that is the best approach